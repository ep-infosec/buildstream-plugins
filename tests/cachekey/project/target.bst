kind: stack

depends:
- sources/bzr1.bst
- sources/cargo1.bst
- sources/cargo2.bst
- sources/docker1.bst
- sources/docker2.bst
- sources/git1.bst
- sources/git2.bst
- sources/patch1.bst
- sources/patch2.bst
- sources/patch3.bst
- sources/pip1.bst
- elements/autotools1.bst
- elements/cmake1.bst
- elements/make1.bst
- elements/meson1.bst
- elements/pip1.bst
