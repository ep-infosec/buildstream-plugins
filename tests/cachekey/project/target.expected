3dc2ed8ab56f9c706e1fbf4bacc1ab1e38d20f66c686f1ab05d5ad0e504272fc